bftengine/include/bftengine/DbCheckpointManager.hpp

```diff
@@ -115,6 +115,11 @@ class DbCheckpointManager {
     if (monitorThread_.joinable()) monitorThread_.join();
   }
   void sendInternalCreateDbCheckpointMsg(const SeqNum& seqNum, bool noop);
+  SeqNum getLastStableSeqNum() const;
+  void setOnStableSeqNumCb_(std::function<void(SeqNum)> cb) { onStableSeqNumCb_ = cb; }
+  void onStableSeqNum(SeqNum s) {
```

Comment:
Do we call this method (I don't see it)?

Reply:
Actually, we only call it in the constructor. Could you please explain the rationale?

Reply:
I was looking in the wrong place, please ignore.